client/lib/abtest/active-tests.js

```diff
@@ -111,10 +111,10 @@ module.exports = {
 	},
 
 	userFirstSignup: {
```

Comment:
I'm wondering if we should use the same name for this test. Would it make sense to use another name, so there is no confusion between the two tests?